packages/react/src/jsx/jsx-local-namespace.ts

```diff
@@ -80,7 +80,11 @@ export namespace CompiledJSX {
        * The class name prop now can be given the output of xcss prop from `@compiled/react`.
        */
       className?: string | Record<string, any> | null | false;
-      css?: CssFunction<void> | CssFunction<void>[];
+      css?:
+        | CssFunction<void>
+        | CssFunction<void>[]
+        | (CssFunction<void> & CompiledPropertyDeclarationReference)
```

Comment:
Ah adding to css prop because it was type erroring?

Reply:
Yeah, exactly, because our `css|cssMap` returns `CompiledStyles = { … } & CompiledPropertyDeclarationReference` (top-level).

I wasn't quite certain if there's any sweeping implications of this, to be frank…LGTM 🤣